.github/workflows/CI.yml

```
@@ -15,20 +15,36 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.9'
-          - 'nightly'
+          - 'lts'
+          - 'pre'
```

```suggestion
          - 'lts'
          - '1'
          - 'pre'
```
`1` here will end up meaning `1.12`, `pre` will be `1.13-alpha2` (I think?)

---

Yes, that is usually what I run (`lts` + `pre`). I guess if it works on both, chances of not running in the current release are low :-)